lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_builtin_method.rb

```diff
@@ -181,12 +184,23 @@ def self.detect_category(obj_name, prov_obj)
       case obj_name
       when "orchestration_stack", "orchestration_stack_retire_task"
         CLOUD
-      when "miq_request", "miq_provision", "vm_migrate_task", "vm_retire_task"
+      when "miq_request"
+        case prov_obj
+        when nil
+          UNKNOWN
```

Comment:
With ```obj_name = 'miq_request'```, ```prov_obj = nil```,  seems old logic would return ```nil```. Is it right?

Reply:
Yes, but I think it's a bug and should be unknown. But you're right, I'll flip back to nil just to be safe.